include/zephyr/drivers/cellular.h

```diff
@@ -29,16 +29,26 @@
 extern "C" {
 #endif
 
-/** Cellular access technologies */
+/** Cellular access technologies (3GPP TS 27.007 AcT) */
 enum cellular_access_technology {
 	CELLULAR_ACCESS_TECHNOLOGY_GSM = 0,
-	CELLULAR_ACCESS_TECHNOLOGY_GPRS,
-	CELLULAR_ACCESS_TECHNOLOGY_UMTS,
-	CELLULAR_ACCESS_TECHNOLOGY_EDGE,
-	CELLULAR_ACCESS_TECHNOLOGY_LTE,
-	CELLULAR_ACCESS_TECHNOLOGY_LTE_CAT_M1,
-	CELLULAR_ACCESS_TECHNOLOGY_LTE_CAT_M2,
-	CELLULAR_ACCESS_TECHNOLOGY_NB_IOT,
+	CELLULAR_ACCESS_TECHNOLOGY_GSM_COMPACT = 1,
+	CELLULAR_ACCESS_TECHNOLOGY_UTRAN = 2,
+	CELLULAR_ACCESS_TECHNOLOGY_GSM_EGPRS = 3,
+	CELLULAR_ACCESS_TECHNOLOGY_UTRAN_HSDPA = 4,
+	CELLULAR_ACCESS_TECHNOLOGY_UTRAN_HSUPA = 5,
+	CELLULAR_ACCESS_TECHNOLOGY_UTRAN_HSDPA_HSUPA = 6,
+	CELLULAR_ACCESS_TECHNOLOGY_E_UTRAN = 7,
+	CELLULAR_ACCESS_TECHNOLOGY_EC_GSM_IOT = 8,
+	CELLULAR_ACCESS_TECHNOLOGY_E_UTRAN_NB_S1 = 9,
+	CELLULAR_ACCESS_TECHNOLOGY_E_UTRA_5G_CN = 10,
+	CELLULAR_ACCESS_TECHNOLOGY_NR_5G_CN = 11,
+	CELLULAR_ACCESS_TECHNOLOGY_NG_RAN = 12,
+	CELLULAR_ACCESS_TECHNOLOGY_E_UTRA_NR_DUAL = 13,
+	CELLULAR_ACCESS_TECHNOLOGY_E_UTRAN_NB_S1_SAT = 14,
+	CELLULAR_ACCESS_TECHNOLOGY_E_UTRAN_WB_S1_SAT = 15,
+	CELLULAR_ACCESS_TECHNOLOGY_NG_RAN_SAT = 16,
+	CELLULAR_ACCESS_TECHNOLOGY_UNKNOWN = 255,
```

Comment:
If you could also do the doxygen comments here this would be great!

Reply:
Done